ml_data_management_code_NB.Rmd

```diff
@@ -0,0 +1,342 @@
+---
+title: "ML Data Management Code"
+output: html_notebook
+---
+# ML Data Management Code
+
+## Intro
+
+This is the R Notebook for our ML Data Management Code. We originally worked on this in an RScript, but will now use an R Notebook for it. The data used in this file will come from a simulated dataset.
+
+## Download packages
```

Comment:
I would change this to Load packages. You download with `install.packages()`. This loads the package for this session.